health-on-time/src/Components/Sidebar.js

```diff
@@ -0,0 +1,135 @@
+import "../Styles/Sidebar.css";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import Image from "react-bootstrap/Image";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+import CalendarPage from "../Pages/CalendarPage";
```

Comment:
It doesn't seem like you need these components CalendarPage or MedicineCabinet from. Since you aren't using them, you can delete those lines.

Reply:
Or maybe I was supposed to use the page components - if they'll be handling layout? Should I switch to them instead of the components for the right hand side?

Reply:
As for profile image, I wasn't sure how it was to be accessed. Please advise.